README.md

```diff
@@ -71,6 +71,11 @@
       <td>
         <a href="/docs/README.tw.md">臺灣</a>
       </td>
+      <td>
```

Comment:
Can you reorganize the table such that there are two rows, one row of `9` and one row of `8? With "Add +" as the last one?

Reply:
oh yes, done in my last commi :)